src/utils/exceptionThrowers.ts

```diff
@@ -36,7 +51,10 @@ export const throwChatSDKError = (chatSDKError: ChatSDKErrors, e: unknown, scena
         console.error(message);
     }
 
-    throw new Error(exceptionDetails.response);
+    throw new CustomChatSDKError(
+        exceptionDetails.response,
+        ((e as any)?.isAxiosError && (e as any)?.response?.status) ? (e as any)?.response?.status : undefined

```

Comment:
We should add `eslint-disable-line @typescript-eslint/no-explicit-any` instead of disabling the rule for the whole file